test/E2E/TestFunctionApp/Profile.ps1

```diff
@@ -0,0 +1,3 @@
+function Get-ProfileString {
+    "PROFILE"
+}
```

Comment:
Can you change the file name to `profile.ps1`?

Reply:
done